org.libreoffice.LibreOffice.json

```diff
@@ -64,8 +64,8 @@
                 },
                 {
                     "type": "archive",
-                    "url": "https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.tar.xz",
-                    "sha256": "cebb705dbbe26a41d359b8be08ec066caba4e8686670070ce44bbf2b57ae113f",
+                    "url": "apache-ant-1.10.9-bin.tar.xz",
+                    "sha256": "42f12ea7dc854b69aac0bd05c5767252dddd297888503e69b8aae0a1dc791cf5",
```

Comment:
Why would you want to do all these changes here?

The use of external project versions here is exactly the same as in the upstream LO version. The list here and the upstream list at <https://git.libreoffice.org/core/+/refs/tags/libreoffice-7.5.1.2/download.lst> are kept in sync with a tool from <https://git.libreoffice.org/core/+/c5f620f9f6abae0213fdda17fae10f281b7bdf9f%5E!/> "add generate-flatpak-manifest.sh to generate FlatPak manifest".

Reply:
oh thanks, sorry for my mistake, i not know that have a list for use specific version

Reply:
> Why would you want to change this?
> 
> The use of external project versions here is exactly the same as in the upstream LO version. The list here and the upstream list at https://git.libreoffice.org/core/+/refs/tags/libreoffice-7.5.1.2/download.lst are kept in sync with a tool from https://git.libreoffice.org/core/+/c5f620f9f6abae0213fdda17fae10f281b7bdf9f%5E!/ "add generate-flatpak-manifest.sh to generate FlatPak manifest".

sorry but the ant is not in this list

Reply:
ant look's to be external jar archive release, is the same in build in my notebook, download a apache ant jar and define the location in path

Reply:
Ant looks to be a independent from Libreoffice lists, apache ant is a jar file, and ant is a build-system for Java code, it's is similar to makefile, can be good use a newest release, it's not break compatiblity.
GVFS have a newest release too, can be good to use this, and Java 17 is not more newest then Java 19 but can be good to use a newest LTS release

Reply:
Yeah, sorry for the confusion on my part.  Ant is indeed a prerequisite that we do not bundle as an external source with LO upstream.
So you could either add your update of Ant back into this PR, or let me know whether you're fine with the current state of the PR (without updated Ant) in which case I'd go ahead and push it.
In any case, sorry for the back and forth, and thanks for the patch!